activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Variants are tracked in the database to avoid existence checks in the storage service.
```

Comment:
Let's discuss the upgrade story, too. That you need to migrate your db to get the new table, turn on `track_variants`, and then it'll just work with no backfilling?

Reply:
I was planning to put that in the upgrading guide when the time came, but I can add it here, too.